tools/gen_ort_dockerfile.py

```diff
@@ -347,15 +361,20 @@ def dockerfile_for_windows(output_file):
 
 WORKDIR /opt/onnxruntime/bin
 RUN copy \\workspace\\build\\Release\\Release\\onnxruntime.dll \\opt\\onnxruntime\\bin
-RUN copy \\workspace\\build\\Release\\Release\\onnxruntime_providers_cuda.dll \\opt\\onnxruntime\\bin
 RUN copy \\workspace\\build\\Release\\Release\\onnxruntime_providers_shared.dll \\opt\\onnxruntime\\bin
```

Comment:
Copy if it exists. onnxruntime_providers_shared.dll is not created when only CPU ep is enabled (note: it will be created if GPU is disabled but Openvino is enabled but not if both are disabled)

Reply:
My bad - this behavior was changed and now it is always built.